File: doc/snippets/whatsapp/message/reply/snippet.js
const url = 'https://api.smsbox.pro/waba/json/<WABAID>/message/<PHONEID>';
const headers = {
    'Authorization': 'App <APIKEY>',
    'Content-Type': 'application/json'
};
const data = {
    recipients: [
        "+336XXXXXXXX"
    ],
    contentMessage: {
        type: "text",
        context: {
            message_id: "wamid.xxxxxxxxxxxxxxxxxxxxxxxxxxxx"
        },
        text: {
            body: "Reply to the previous message."
        }
    }
};

fetch(url, {
    method: 'POST',
    headers: headers,
    body: JSON.stringify(data)
})
    .then(response => response.json())
    .then(data => console.log(data))
    .catch(error => console.error('Error:', error));

File: doc/snippets/whatsapp/message/reply/snippet.py
import requests
import json

url = 'https://api.smsbox.pro/waba/json/<WABAID>/message/<PHONEID>'
headers = {
    'Authorization': 'App <APIKEY>',
    'Content-Type': 'application/json'
}
data = {
    "recipients": [
        "+336XXXXXXXX"
    ],
    "contentMessage": {
        "type": "text",
        "context": {
            "message_id": "wamid.xxxxxxxxxxxxxxxxxxxxxxxxxxxx"
        },
        "text": {
            "body": "Reply to the previous message."
        }
    }
}

response = requests.post(url, headers=headers, data=json.dumps(data))

print(response.status_code)
print(response.json())
